libraries/abstractions/common_io/test/test_iot_sdio.c

```diff
@@ -0,0 +1,1073 @@
+
+        gIotSdioHandle[ testIotSdio_HOST_INSTANCE ][ testIotSdio_SLOT_INSTANCE ] = NULL;
+    }
+}
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Tear down function called after each test in this group is executed.
+ */
+TEST_TEAR_DOWN( TEST_IOT_SDIO )
+{
+}
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief user callback function
+ */
+void prvSdioCallback( IotSdioSlotHandle_t const pxSdioHandle,
+                      IotSdioEventType_t eSdioEvent, void * pvUserContext )
+{
+    switch (eSdioEvent)
+    {
+        case eSdioCardInterruptEvent:
+            break;
+        case eSdioCardInsertedEvent:
+            break;
+        case eSdioCardRemovedEvent:
+            break;
+        case eSdioCardPowerOnEvent:
+            break;
+        case eSdioCardPowerOffEvent:
+            break;
+        default:
+            break;
+    }
+}
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Function to define which tests to execute as part of this group.
+ */
+TEST_GROUP_RUNNER( TEST_IOT_SDIO )
+{
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioOpenClose );
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioDoubleOpenDoubleClose );
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioOpenCloseFuzzing );
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioCardConnectDisconnect );
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioCardConnectDisconnectFuzzing );
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioWriteReadDirect );
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioWriteReadDirectFuzzing );
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioWriteReadExtended );
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioWriteReadExtendedFuzzing );
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioIoctl );
+    RUN_TEST_CASE( TEST_IOT_SDIO, AFQP_IotSdioIoctlFuzzing );
+}
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Test Function to test iot_sdio_open and iot_sdio_close.
+ *
+ */
+TEST( TEST_IOT_SDIO, AFQP_IotSdioOpenClose )
+{
+    IotSdioSlotHandle_t xSdioHandle = NULL;
+
+    int32_t lRetVal;
+
+    /* Open with valid host index and slot index should return a non NULL handle. */
+    xSdioHandle = iot_sdio_open(uctestIotSdioValidHostIdx,
+                                uctestIotSdioValidSlotIdx);
+    TEST_ASSERT_NOT_EQUAL( NULL, xSdioHandle );
+
+    /* Closing a valid handle should return success. */
+    lRetVal = iot_sdio_close(xSdioHandle);
+    TEST_ASSERT_EQUAL(IOT_SDIO_SUCCESS, lRetVal);
+}
+
+/**
+ * @brief Test Function to test double open and double close.
+ *
+ */
+TEST( TEST_IOT_SDIO, AFQP_IotSdioDoubleOpenDoubleClose )
+{
+    IotSdioSlotHandle_t xSdioHandle = NULL;
+    IotSdioSlotHandle_t xSdioHandle2 = NULL;
+
+    int32_t lRetVal;
+
+    /* Open with valid host index and slot index should return a non NULL handle. */
+    xSdioHandle = iot_sdio_open(uctestIotSdioValidHostIdx,
```

Comment:
Can you run uncrustify on these two files, it does not conform to style guide

Reply:
done